chart/values.yaml

```diff
@@ -647,6 +647,8 @@ jupyterhub:
     pod-anti-affinity-preferred: []
     pod-anti-affinity-required: []
     startNotebookConfigMap: start-notebook-d
+    SPAWNER_START_TIMEOUT: 300
+    SPAWNER_HTTP_TIMEOUT: 30
```

Comment:
sorry, forget to mention it, we prefer the camel case.

* spawnerStartTimeout
* spawnerHttpTimeout

Reply:
sure, updated it accordingly!